taraxa/state/dpos/tests/dpos_test_utils.go

```diff
@@ -131,15 +131,15 @@ func (self *DposTest) execute(from common.Address, value uint64, input []byte) v
 		Input: input,
 	})
 
-	self.st.EndBlock(nil)
+	self.st.EndBlock(nil, nil, nil)
 	self.st.Commit()
 	return res
 }
 
 func (self *DposTest) AddRewards(rewards map[common.Address]*big.Int) {
 	self.blk_n++
-	self.st.BeginBlock(&vm.BlockInfo{}, rewards)
-	self.st.EndBlock(nil)
+	self.st.BeginBlock(&vm.BlockInfo{})
+	self.st.EndBlock(nil, nil, nil)
```

Comment:
what about the tests now?

Reply:
what about tests ?

Reply:
they will stop working as you are not passing rewards map anywhere

Reply:
some go tests or which ones ? C++ tests work

Reply:
yes go tests https://github.com/Taraxa-project/taraxa-node/issues/1749

Reply:
fixed